src/components/clusterConfiguration/ReviewValidations.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import _ from 'lodash';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import {
+  global_success_color_100 as okColor,
+  global_danger_color_100 as dangerColor,
+} from '@patternfly/react-tokens';
+import {
+  ValidationsInfo as ClusterValidationsInfo,
+  ValidationGroup as ClusterValidationGroup,
+  Validation as ClusterValidation,
+} from '../../types/clusters';
+import {
+  ValidationsInfo as HostValidationsInfo,
+  ValidationGroup as HostValidationGroup,
+  Validation as HostValidation,
+} from '../../types/hosts';
+import ClusterWizardContext from '../clusterWizard/ClusterWizardContext';
+import { ClusterWizardStepsType, findValidationFixStep } from '../clusterWizard/wizardTransition';
+import { wizardStepNames } from '../clusterWizard/ClusterWizardStep';
+import { Cluster, Host, stringToJSON } from '../../api';
+
+const AllValidationsPassed = () => (
+  <>
+    <CheckCircleIcon color={okColor.value} /> All validations passed
+  </>
+);
+
+const PendingValidations: React.FC<{ id: string; count: number }> = ({ id, count }) => (
+  <div id={id}>
+    There {count === 1 ? 'is' : 'are'} still {count} pending validation
+    {count === 1 ? '' : 's'}.
+  </div>
+);
+
+const ValidationActionLink: React.FC<{ step: ClusterWizardStepsType }> = ({ step }) => {
+  const { setCurrentStepId } = React.useContext(ClusterWizardContext);
+  return (
+    <Button variant={ButtonVariant.link} onClick={() => setCurrentStepId(step)} isInline>
+      {wizardStepNames[step]}
+    </Button>
+  );
+};
+
+const FailingValidation: React.FC<{
+  validation: HostValidation | ClusterValidation;
+  clusterGroup?: ClusterValidationGroup;
+  hostGroup?: HostValidationGroup;
+}> = ({ validation, clusterGroup, hostGroup }) => {
+  const issue = `Checking of ${_.lowerFirst(validation.message).replace(/\.$/, '')}, failed. `;
```

Comment:
IMO it would be better to use HOST_VALIDATION_LABELS[validation.id] (and define labels for cluster validations too) as current approach does not work with all validation messages. e.g.:
![Screenshot from 2021-02-18 15-04-23](https://user-images.githubusercontent.com/1121740/108370245-b78b7780-71fc-11eb-9ef7-1d8996bbdcf4.png)

Reply:
I wanted to avoid that due to the need of subsequent maintenance. But you are right, we will probably not able to provide good wording this way.

Reply:
done